com/abweichung.go

```diff
@@ -1,9 +1,24 @@
 package com
 
-import "github.com/hochfrequenz/go-bo4e/enum/abweichungsgrund"
+import (
+	"github.com/go-playground/validator/v10"
+	"github.com/hochfrequenz/go-bo4e/enum/abweichungsgrund"
+)
 
 type Abweichung struct {
 	Referenz                  string                             `json:"referenz,omitempty"`
 	AbweichungsGrund          *abweichungsgrund.AbweichungsGrund `json:"abweichungsgrund,omitempty" validate:"omitempty"`
-	AbweichungsGrundBemerkung string                             `json:"abweichungsgrundBemerkung,omitempty"`
+	AbweichungsGrundBemerkung *string                            `json:"abweichungsgrundBemerkung,omitempty"`
+	AbweichungsgrundCode      *string                            `json:"abweichungsgrundCode,omitempty" validate:"omitempty"`
+	AbweichungsgrundCodeliste *string                            `json:"abweichungsgrundCodeliste,omitempty" validate:"omitempty"`
+}
+
+// AbweichungStructLevelValidation does a cross check on a Abweichung object
+func AbweichungStructLevelValidation(sl validator.StructLevel) {
+	// ToDo: use required_without/required_if instead of own validator
```

Comment:
warum machen wir das nicht einfach? :D
ich habe es nicht getestet aber würde denken, dass meine suggestion oben in die richtige richtung geht.
oder ist der plan, dass man die validation händisch anschalten muss?

Reply:
s.o.